src/components/Article.jsx

```diff
@@ -1,23 +1,37 @@
 import "../styles/Article.css"
 import { useEffect, useState } from "react"
 import { useParams } from "react-router"
-import { getSingleArticle } from "../utils/fetches"
+import { getSingleArticle, patchArticleVotes } from "../utils/fetches"
 
 const Article = () => {
   const { article_id } = useParams()
   const [article, setArticle] = useState({})
+  const [votes, setVotes] = useState(0)
   const [isLoading, setIsLoading] = useState(true)
 
   useEffect(() => {
     getSingleArticle(article_id).then(({ article }) => {
       const date = new Date(article.created_at)
       article = { ...article, created_at: date.toUTCString() }
       setArticle(article)
+      setVotes(article.votes)
       setIsLoading(false)
     })
   }, [])
 
-  if (isLoading) return <>loading...</>
+  const sendVote = (num) => {
+    setVotes((a) => a + num)
+    patchArticleVotes(article_id, num)
+      .then(({ article }) => {
+        setArticle(article)
+        setVotes(article.votes)
+      })
+      .catch(() => {
```

Comment:
This is nice, could there be some feedback to the user as to why the number is going back? I know why when I turn the internet off on browser, but would a user?